<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copas_pedia_widget</class>
 <widget class="QWidget" name="copas_pedia_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CopasPedia</string>
  </property>
  <widget class="QTextEdit" name="content_text_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>591</width>
     <height>321</height>
    </rect>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_line_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QComboBox" name="lang_combo_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="page_combo_box">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
